proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+We propose a European funding for the development of progressive, democratic and/or educational forms of video games, interactive adventures, Web documentaries and other innovative forms of storytelling. 
+See for instance: Jeu d'influences, Hackstage Pass, Papers Please, Fort McMoney
+
+# 8 
+# 04.07.2018 Tech Pillar Suggestions DSC Lower Austria - 18-07-04
+
+My input is grouped in chapters which me thinks should be part of the Tech Pillar.
+Most emphasis should be given to consequences of tech developments for average 
+people rather than ideas to develop new technologies and supporting nerds.
+
+Surveillance capitalism and its economic and political implications
+
+- explain mechanisms of data collection, data exchange, contributing companies like  Oracle, Acxiom, Google, Microsoft, Facebook, banks, insurances, telecoms ...
+- how are collected data used in advertising, risk assessment, customer management, ...
+- what are the negative economic consequences for users of pervasive data collection
+
+A wide range of data and analytics companies in marketing, customer management and risk analytics seamlessly gather, analyze, share, and trade consumer data as well as combine it with further information from thousands of other companies. While the data and analytics industry provides the means to deploy these powerful technologies, businesses in many industries equally contribute both to intensifying the amount and detail of the collected data and the ability to utilize it. An excellent overview about methods and implication of a developing society of pervasive digital social control was compiled by Wolfie Christl and can be found here:
+http://crackedlabs.org/en/corporate-surveillance/
+
+This report finds that the networks of online platforms, advertising technology providers, data brokers, and other businesses can now monitor, recognize, and analyze individuals in many life situations. Information about individuals’ personal characteristics and behaviors is linked, combined, and utilized across companies, databases, platforms, devices, and services in real-time. With the actors guided only by economic goals, a data environment has emerged in which individuals are constantly surveyed and evaluated, categorized and grouped, rated and ranked, numbered and quantified, included or excluded, and, as a result, treated differently.
+
+- surveillance capitalism has developed meanwhile massive political implications 
+This is nicely described in 'Cory Doctorow: Zuck’s Empire of Oily Rags': 
+http://locusmag.com/2018/07/cory-doctorow-zucks-empire-of-oily-rags/
+
+Given enough surveillance, companies can sell us anything: Brexit, Trump, ethnic 
+cleansing in Myanmar, and successful election bids for absolute bastards like Turkey’s Erdogan and Hungary’s Orban.
+
+- develop a political strategy against surveillance resp its implications and usage some  ideas: 
+- break up integrated enterprises like Google, Facebook, Oracle, advertising 
```

Comment:
Breaking up the monopolists would need to be repeated yearly as the digital market tends to recreate new monopolists. The best way to prevent trading of data is to not let anyone *have* any data to trade — otherwise trade will happen, either legally or in the darknet, simply because the data is there and there is no evidence of it being abused or stolen. - lynx